information/specification.md

```diff
@@ -0,0 +1,706 @@
+    list](#requested-output-list).
+
+    Normally, this parameter would be assigned with the complete [UTxO
+    set](#utxo-set) of a wallet, giving the coin selection algorithm access to
+    the total value associated with that wallet.
+
+ 3. #### Maximum Input Count
+
+    An _upper bound_ on the number of UTxO entries that the coin selection
+    algorithm is permitted to select from the [initial UTxO
+    set](#initial-utxo-set).
+
+## Results
+
+A coin selection function produces the following result values:
+
+ 1. #### Coin Selection
+
+    The _coin selection_ is a record with three fields:
+
+      * A set of **_inputs_**, equivalent to a subset of the
+        [initial UTxO set](#initial-utxo-set).
+
+        Represents the value that has been selected from the wallet in order to
+        cover the total payment value.
+
+      * A set of **_outputs_** (see [transaction output](#transaction-output)).
+
+        Represents the set of payments to be made to recipient addresses.
+
+      * A set of **_change values_**, where each change value is simply a
+        [coin amount](#coin-amount).
+
+        Represents the change to be returned to the wallet.
+
+ 2. #### Remaining UTxO Set
+
+    The _remaining UTxO set_ is a subset of the [initial UTxO
+    set](#initial-utxo-set).
+
+    It represents the set of values that remain after the coin selection
+    algorithm has removed values to pay for entries in the [requested output
+    list](#requested-output-list).
+
+    If a coin selection algorithm is applied to the _complete_ UTxO set of a
+    wallet, then the remaining UTxO set represents the _updated_ UTxO set of the
+    wallet.
+
+## Properties
+
+### Coverage of Payments
+
+This property states that the total value of the [coin
+selection](#coin-selection) result is sufficient to _cover_ the total value of
+the [requested output list](#requested-output-list).
+
+In particular:
+
+  * V<sub>_selected_</sub> ≥ V<sub>_requested_</sub>
+
+Where:
+
+  * V<sub>_requested_</sub>
+
+    is the total value of the [requested output list](#requested-output-list)
+
+
+  * V<sub>_selected_</sub>
+
+    is the total value of the _inputs_ field of the [coin
+    selection](#coin-selection) result.
+
+### Correctness of Change
+
+This property states that the correct amount of _change_ was generated.
+
+In particular:
+
+  * V<sub>_selected_</sub> = V<sub>_requested_</sub> + V<sub>_change_</sub>
+
+Where:
+
+  * V<sub>_change_</sub>
+
+    is the total value of the _change_ field of the [coin
+    selection](#coin-selection) result.
+
+  * V<sub>_requested_</sub>
+
+    is the total value of the [requested output list](#requested-output-list)
+
+  * V<sub>_selected_</sub>
+
+    is the total value of the _inputs_ field of the [coin
+    selection](#coin-selection) result.
+
+### Conservation of UTxO
+
+This property states that the [initial UTxO set](#initial-utxo-set) is
+_conserved_ in the [results](#results).
```

Comment:
This seems to be a rewording of "conservation of UTxO".

Maybe say something like "This property states that every UTxO in the initial set is included in either the coin selection inputs, or in the remaining utxo set, but not both."

Reply:
> Maybe say something like "This property states that every UTxO in the initial set is included in either the coin selection inputs, or in the remaining utxo set, but not both."

Thanks, this wording is very clear and concise!

Fixed in e2a6a6ad2f50bc1f79b70a35b7bcfb5a82c2058b.